Firmware/MVP/LTC68042.cpp

```diff
@@ -242,10 +275,10 @@ void printCellVoltage_max_min()
     if( maxCellVoltage > highestCellVoltage )
     {
       highestCellVoltage = maxCellVoltage;
-      lcd2.print(" (max:");
-      lcd2.print( (highestCellVoltage * 0.0001) , 3);
-      lcd2.print(")");
     }
```

Comment:
I moved these blocks out of the if statements because I saw they would get cleared by the splash screen

Reply:
Writing to the 4x20 display is fairly slow.  Hence I only refresh the actual numerical data.
I worry that sending too much data to the screen every cycle is eventually going to be a problem.  See my notes further down for potential compounding issues.

Reply:
okay what I'll do is add a bool for this function to allow force printing of these numbers, such as on screen wake